packages/cli/examples/mask.ts

```diff
@@ -0,0 +1,89 @@
+/*
+ * This file was generated with ❤️ by wasm.glass and is licensed
+ * for you under WTFPL OR 0BSD OR Unlicense OR MIT OR BSD-3-Clause.
+ * Note that different terms apply for the contract's source code and schema.
+ * Type generation powered by json-schema-to-typescript.
+ */
```

Comment:
I'd remove this header, since you do much more in this file and it can be misleading this way.